packages/next-on-pages/docs/supported.md

```diff
@@ -28,11 +24,23 @@ The [Cloudflare workers runtime supports certain Node.js APIs](https://developer
 - `util`
 - `async_hooks`
 
-### External Packages
+## External Packages
+
+You are free to use any external npm package with your Next application as long as it doesn't:
+
+- require unsupported Node.js APIs (see above),
+- use [JavaScript APIs](https://developers.cloudflare.com/workers/runtime-apis/web-standards/#javascript-standards) disabled by Cloudflare due to security concerns,
+- nor it relies on features not supported by `@cloudflare/next-on-pages` (see below)
```

Comment:
I think this is a triple negative...  "as long as it doesn't" + "nor" + "not supported".

how about:

"- relies on Next features not suported by `@cloudflare/next-on-pages` (see below)"

Reply:
yeah good point 😅 , actually we have `require`, `use` and `relies`... I think this can be simplified further by just putting the verb in the prefix sentence 🙂